src/preact.d.ts

```diff
@@ -99,6 +99,17 @@ declare namespace preact {
 		forceUpdate(callback?: () => void): void;
 
 		abstract render(props?: RenderableProps<P>, state?: Readonly<S>, context?: any): ComponentChild;
+
+		// Add these variables to avoid descendants shadowing them
+		private __key;
```

Comment:
In the Preact codebase, any properties with a leading underscore get minified using the mapping from here:
https://github.com/developit/preact/blob/master/config/properties.json

In addition, `next*` and `prev*` are also minified.  We might as well add them all if we're looking to warn folks.

Reply:
Good catch. I'll add them as well